contents/spotlight/startup-krea.md

```diff
@@ -0,0 +1,54 @@
+---
+title: How AI startups can deal with massive data, with Diego from Krea.ai
+date: 2025-01-24
+author:
+  - kevan-gilbert
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/413293711_2ad62f5e_8fca_44cf_b73a_0848f9537a56_3e39fca48c.jpg
+featuredImageType: full
+category: Startups
+---
+The irony of going viral is that everybody wants it, but nobody’s ready for it. 
+
+There is often such a sudden increase in data usage that servers strain, performance degrades and costs can shoot up for founding teams.
+
+The team at [Krea](https://www.krea.ai/) surfed that wave — when their app went viral in Southeast Asia, hitting 200,000 sign-ups in a single day. Having already [set up PostHog to track events](/product-analytics#installation) for Krea, they were ready in advance for such a surge.
+
+Diego Rodriguez, cofounder at Krea, senses that startups usually don't need to care about big amounts of data until it’s too late but that way of working may change soon if it didn't do it already.
+
+His advice? If you’re building an AI-first app, you’re going to
+ need to be ready for massive surge of data, independently of whether virality is in the equation or not. Here’s how founders can prepare with a stronger foundation.
+
+## 1. AI-first apps generate more data than you can dream
+
+Generative AI presents an entirely different paradigm for managing data such as events. “When you track user events on a finance website,” Diego says. “People don't go to a website and spend money a thousand times on a visit. That just doesn't happen. But people do come to Krea and they may make 1,000 images in a single session.”
+
+Diego explains: “Let’s say your product is Canva. How much data is passing through your product? A user creates one artifact: let’s say, a poster. There it is. They’ll create one poster. You don't make many posters per second, or many posters within one minute. You make *A* poster. You work on it, and then you finish and then you save it. But in the world of AI?
+
+For me, the equivalent of an artifact that the user made it is not a poster — it’s a frame that you make on the [Krea Realtime](https://www.krea.ai/apps/image/realtime) tool.  But the thing is, within one minute you can make a _lot_ of images, right? Yet, each frame is an artifact of a kind — one moment or “event” in terms of product analytics. The person is making images, but suddenly you are not processing few of them in a few seconds; instead, you are processing many frames per second during, say, 20 minutes. We are not talking about percentage differences anymore—it's in the orders of magnitude.”
```

Comment:
This is a great point, but I'm not sure we're leading it anywhere. It needs a conclusion or lesson at the end. 

E.g.: "Generative AI creates data at an unprecedented scale _so you can't use standard analytics tools which don't cater to the scale of AI_" or "Generative AI creates data at an unprecedented scale _so you need to be smart about what you track and here's how we instrumented our data capture_"

Otherwise it just peters out at "AI makes a lot of data" - which is true, but not interesting